app/lib/month.rb

```diff
@@ -66,9 +66,11 @@ def self.current
   def self.parse_month(month)
     return month if month.is_a?(Month)
 
+    raise ArgumentError unless /^\d{4}-\d{2}$/.match?(month)
```

Comment:
I'd say keep the `ArgumentError` for consistency, as you point in the comment @mayorova.

Reply:
we handle error condition at the same place

Reply:
Code Climate was complaining about code complexity, I guess because of throwing and catching. So, I changed it. Not sure though, I like it less, as I think it's more confusing.

Reply:
I agree with you, I find it more confusing now than before. I'd vote for keeping the previous version.